### PR TITLE
#140 visual indication of which platform instructions being shown, is…

### PR DESCRIPTION
**Pull request by gunjanmimo**
… added

Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

**Comment by gunjanmimo** [comment]
@leecalcote sir, may you check the pr now?

File: _includes/getting-started.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
     <div id="getting-started">
     <div style="margin-top: auto;" id="box"> 
       <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
-        with <span>Meshery</span> in<br>
+        with Meshery in<br>
```
Comment (kushthedude) [inline _includes/getting-started.html:12]: No need to remove the span here

File: _includes/getting-started.html [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
         Easy Steps<br>
-      <p style="font-size: 20px;">See how easy it is to get up<br>and running with <span>Meshery</span>, right away.</p></h1>
+      <p style="font-size: 20px;">See how easy it is to get up<br>and running with Meshery, right away.</p></h1>
```
Comment (kushthedude) [inline _includes/getting-started.html:14]: Same no need to remove span

File: index.html [hunk 3]
```diff
@@ -96,7 +96,15 @@ <h2>Interoperate <strong>multiple</strong> service meshes</h2>
 						</tr>
 						<tr>
 							<td><a href="https://github.com/layer5io/meshery-octarine">
+<<<<<<< HEAD
+<<<<<<< HEAD
+								<img width="30px" src='/images/oct.svg' alt='Octarine Service Mesh' class="adapter-logo">Meshery adapter for Octarine</a> 
+=======
 								<img src='images/octarine.png' alt='Octarine Service Mesh' class="adapter-logo">Meshery adapter for Octarine</a> 
+>>>>>>> upstream/master
```
Comment (kushthedude) [inline index.html:104]: Fix this

File: index.html [hunk 4]
```diff
@@ -96,7 +96,15 @@ <h2>Interoperate <strong>multiple</strong> service meshes</h2>
 						</tr>
 						<tr>
 							<td><a href="https://github.com/layer5io/meshery-octarine">
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (kushthedude) [inline index.html:100]: Fix this too

**Comment by kushthedude** [comment]
Also remove the commented code.

File: _includes/getting-started.html (hunk 1 above)
Comment (leecalcote) [reply _includes/getting-started.html:12]: It's an errant tag, though, right? Residual from your removal of the blue highlight. I wonder if there's value in leaving it.

File: index.html (hunk 4 above)
Comment (leecalcote) [reply index.html:100]: Ouch, yes, for certain.

**Comment by gunjanmimo** [comment]
Thank you @lee sir.

**Pull request merged**